diff --git a/node_modules/@types/react-email-editor/index.d.ts b/node_modules/@types/react-email-editor/index.d.ts
index e0078bf..1a425ea 100644
--- a/node_modules/@types/react-email-editor/index.d.ts
+++ b/node_modules/@types/react-email-editor/index.d.ts
@@ -157,11 +157,14 @@ export type FileUploadDoneCallback = (data: FileUploadDoneData) => void;
 
 export type DisplayConditionDoneCallback = (data: DisplayCondition | null) => void;
 export type DisplayConditionCallback = (data: DisplayCondition | EmptyDisplayCondition, done: DisplayConditionDoneCallback) => void;
+export type DoneFunc = (data: {url: string})=>void;
+export type Callback = (data: any,done: DoneFunc) => void;
 
 export default class Component extends ReactComponent<EmailEditorProps> {
     private unlayerReady(): void;
-    registerCallback(type: 'image', callback: FileUploadCallback): void;
-    registerCallback(type: 'displayCondition', callback: DisplayConditionCallback): void;
+    //registerCallback(type: 'image', callback: FileUploadCallback): void;
+    //registerCallback(type: 'displayCondition', callback: DisplayConditionCallback): void;
+    registerCallback(type:'selectImage',callback: Callback): void
     addEventListener(type: string, callback: EventCallback): void;
     loadDesign(design: Design): void;
     saveDesign(callback: SaveDesignCallback): void;
